stories/core_components/GeoPushpinChart.tsx

```diff
@@ -0,0 +1,66 @@
+// (C) 2019-2020 GoodData Corporation
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { screenshotWrap } from "@gooddata/test-storybook";
+
+import { GeoPushpinChart } from "../../src";
+import { onErrorHandler } from "../mocks";
+import {
+    ATTRIBUTE_LOCATION_GEOCHART,
+    MEASURE_SIZE_GEOCHART,
+    MEASURE_COLOR_GEOCHART,
+    ATTRIBUTE_SEGMENT_GEOCHART,
+    ATTRIBUTE_TOOLTIP_GEOCHART,
+} from "../data/geoChartComponentProps";
+
+const wrapperStyle = { width: 800, height: 500 };
+
+storiesOf("Core components/GeoPushpinChart", module)
+    .add("with location, size", () =>
```

Comment:
please add story of location and color

Reply:
@dlcd2203 , please add one more case with filter out some locations